cypress/fixtures/events.json

```diff
@@ -20,7 +20,7 @@
     "image_url": "https://test.event.org/image",
     "streaming_url": "https://test.event.org/video",
     "start_at": "2022-01-01T00:01",
-    "ends_at": "2022-01-02T00:02"
+    "ends_at": "2099-12-31T00:02"
   },
   "eventThree": {
```

Comment:
Maybe we can turn `eventTwo` to `runningEvent`, and `eventThree` to `upcomingEvent`, to make it easier in the future